x-pack/metricbeat/module/aws/elb/_meta/data.json

```diff
@@ -0,0 +1,63 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "aws": {
+        "cloudwatch": {
+            "dimensions": {
+                "LoadBalancerName": "tf-lb-20171218185946539900000004"
+            },
+            "namespace": "AWS/ELB"
+        },
+        "metrics": {
+            "EstimatedALBActiveConnectionCount": {
```

Comment:
I guess default config can now be updated after merging https://github.com/elastic/beats/pull/12840? I hope we can be picky about the statistics we retrieve, limiting it only to the ones that make sense / we plan to use